govcd/system_test.go

```diff
@@ -88,6 +95,7 @@ func (vcd *TestVCD) Test_CreateOrg(check *C) {
 	err = org.Delete(true, true)
 	check.Assert(err, IsNil)
 	// Check if org still exists
+	time.Sleep(10 * time.Second)
```

Comment:
This is a comment regarding all time.Sleep(...) calls in this PR. Even though they are only in tests, we don't want to block code with specific time lengths. This can lead to (a) slower test runs when you we are waiting longer than we should or (b) race conditions when we are waiting not long enough.

What about replacing these sleeps with a handy wait-for function? It should loop by trying to get the needed resource until it's available. For looping, 1 second interval should be a good middle ground.

Reply:
Fixed